service/security/certificate/internal/src/main/java/org/eclipse/kapua/service/certificate/internal/CertificateServiceImpl.java

```diff
@@ -154,4 +157,24 @@ public Certificate generate(CertificateGenerator generator) throws KapuaExceptio
     public List<Certificate> findAncestorsCertificates(KapuaId scopeId, CertificateUsage usage) {
         throw new UnsupportedOperationException();
     }
+
+    @Override
+    public CertificateDomain getServiceDomain() {
+        throw new UnsupportedOperationException();
+    }
+
+    @Override
+    public KapuaTocd getConfigMetadata() throws KapuaException {
+        throw new UnsupportedOperationException();
+    }
+
+    @Override
+    public Map<String, Object> getConfigValues(KapuaId scopeId) throws KapuaException {
```

Comment:
`getConfigMetadata()` must return something, otherwise when fetching available configurations from console form the `Settings` > `AccountSetting` view an error will occur.

This view looks for all configurable services and loads the values per account basis. 

I think that we can make this return a empty map.